admin/src/app/app.module.ts

```diff
@@ -1,33 +1,43 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
 import { ImageCropperModule } from 'ngx-image-cropper';
 import { FormsModule } from '@angular/forms';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { GalleryComponent } from './gallery/gallery.component';
 import { UploadCardComponent } from './upload-card/upload-card.component';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { AdminComponent } from './admin/admin.component';
+import { ReviewCardsListComponent } from './review-cards-list/review-cards-list.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { MatGridListModule } from '@angular/material/grid-list';
 import { ScrollingModule } from '@angular/cdk/scrolling';
 import { HttpClientModule } from '@angular/common/http';
 import {MatTabsModule} from '@angular/material/tabs';
 import {MatTooltipModule} from '@angular/material';
 import { ForbiddenComponent } from './forbidden/forbidden.component';
 
+const appRoutes: Routes = [
```

Comment:
maybe it should be at app-routing.module.ts?

Reply:
it must be removed. thanks!